### ISSUE TITLE
Shapes exports to SVG without their identifiers

### ISSUE THREAD

**Issue by evgeny-sureev** [exyte/Macaw#407]
The "id" attribute is lost after exporting from Macaw to SVG.

**Comment by f3dm76** [comment]
Could you please provide an example?

**Comment by evgeny-sureev** [comment]
Consider this SVG:

```xml
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="1005"
     height="640">
        <rect height="55" x="894" y="259" width="100" fill="#1ABD9D"
              id="dc1b6550-5770-4f44-8173-8449267b8fdf"/>
        <rect height="55" x="894" y="143" width="100" fill="#1ABD9D"
              id="bd7ae432-e619-42d4-8b36-07541e5b2da1"/>
</svg>
```

Now if we import svg into Macaw, we can access identifiers via `tag` property:

```swift
let svg = try SVGParser.parse(text: String(contentsOf: svgData))!
let rect = svg.contents[0]
print(rect.tag.first)
// prints 'dc1b6550-5770-4f44-8173-8449267b8fdf'
```

But after exporting to SVG back, all identifiers are lost:

```swift
print(SVGSerializer.serialize(node: svg))
```

```xml
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="1005"
     height="640">
        <rect height="55" x="894" y="259" width="100" fill="#1ABD9D"/>
        <rect height="55" x="894" y="143" width="100" fill="#1ABD9D"/>
</svg>
```

**Comment by ystrot** [comment]
Hi Evgeny,

This issue fixed in the master branch and quite soon will be available in the 0.9.2 release.
